src/global_const.js

```diff
@@ -1264,36 +1263,41 @@ var skillAmounts = {
     "magnaHakai": {
         "S": [2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5, 2.5],
     },
-    // 楚歌 効果が1本のみとなる為、通常・マグナ攻刃とは別途定義
+    // Since it is only one Soka effect, it is usually defined separately from Magna attacking blade
     "normalSoka": {
         "M": [3.0, 3.0, 4.0, 4.0, 5.0, 5.0, 6.0, 6.0, 7.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0],
     },
     "magnaSoka": {
         "M": [3.0, 3.0, 4.0, 4.0, 5.0, 5.0, 6.0, 6.0, 7.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0, 8.0],
     },
-    // 先制スキルは加護が乗らない　Slv11~14までは仮入力
+    // Pre-emptive skills
```

Comment:
Important info in this comment was:
- Slv 11..14 are temporary values.

means those values are not verified, it was outdated comment ? or leave it.
looks "sensei" has values in slv 11..14 if they are verified then ok. can remove the tail of comment.

Reply:
Yeah already confirmed, I check the skill values from time to time if something changed and Pre-emptive was confirmed some time ago, critical or something was the last thing I updated with updated wiki values when I remembered right.

Reply:
:+1: ok, I did not know the value details, so just a confirmation.